### PR TITLE
add 1.20 exceptions.yaml

### PR DESCRIPTION
**Pull request by kikisdeliveryservice**
Starting the exceptions.yaml file for 1.20 Release.

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]

File: releases/release-1.20/exceptions.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Exceptions requested in 1.20
+# Google Group: https://groups.google.com/forum/#!forum/kubernetes-sig-release
+# Release Team Lead: Kirsten Garrison (@kikisdeliveryservice)
```
Comment (justaugustus) [inline releases/release-1.20/exceptions.yaml:3]: This should match the RT Leads and Lead Shadows.

Example: https://github.com/kubernetes/sig-release/blob/db606943bb342c1ab55cbc4b7809a6e7cb6ebd6d/releases/release-1.19/exceptions.yaml#L3-L4

**Review by justaugustus — CHANGES_REQUESTED** [review]
@kikisdeliveryservice -- I think there are more exceptions approved from this cycle since this PR was created.

Outside of that, one nit on the Leads listing.

File: releases/release-1.20/exceptions.yaml (hunk 1 above)
Comment (kikisdeliveryservice) [reply releases/release-1.20/exceptions.yaml:3]: Oh gosh DUH. Was rushing will update. 🤦

**Comment by kikisdeliveryservice** [comment]
OK finally updated :smile: 

@justaugustus PTAL

/hold cancel

**Pull request merged**